cegs_portal/search/templates/search/v1/experiment.html

```diff
@@ -734,6 +757,11 @@
                 .catch((err) => console.log(err));
         })
         .catch((err) => console.log(err));
+
+    var dropdown = document.getElementById("dropdown");
```

Comment:
Don't forget that the analysis selector may not exist. You have to check for that condition here.